sql/core/src/test/resources/sql-tests/results/hll.sql.out

```diff
@@ -66,9 +70,10 @@ SELECT hll_sketch_estimate(hll_union_agg(sketch, true))
           SELECT hll_sketch_agg(col, 20) as sketch
             FROM VALUES (1) AS tab(col))
 -- !query schema
-struct<hll_sketch_estimate(hll_union_agg(sketch, true)):bigint>
+struct<>
 -- !query output
-1
+java.lang.NoClassDefFoundError
+Could not initialize class org.apache.datasketches.memory.internal.BaseWritableMemoryImpl
```

Comment:
Seems like the output is invalid ..

Reply:
@HyukjinKwon I have regenerated hll.sql.out